frontend/packages/kubevirt-plugin/src/components/vms/vm-users.tsx

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { StatusItem } from '@console/shared/src/components/dashboard/status-card/AlertItem';
+import { BlueInfoCircleIcon } from '@console/shared/src/components/status';
+import { fromNow } from '@console/internal/components/utils/datetime';
+import { getName, getNamespace } from '@console/shared/src/selectors/common';
+import { Table, TableRow, TableData } from '@console/internal/components/factory';
+import {
+  withDashboardResources,
+  DashboardItemProps,
+} from '@console/internal/components/dashboard/with-dashboard-resources';
+import { VMIKind } from '../../types';
+import './vm-resource.scss';
+import { VMStatusBundle } from '../../statuses/vm/types';
+import { VMStatus } from '../../constants/vm/vm-status';
+import { Timestamp } from '@console/internal/components/utils';
+import { isGuestAgentInstalled } from '../dashboards-page/vm-dashboard/vm-alerts';
+
+const tableColumnClasses = [
+  classNames('col-lg-3', 'col-md-3', 'col-sm-4', 'col-xs-6'),
+  classNames('col-lg-3', 'col-md-3', 'col-sm-4', 'hidden-xs'),
+  classNames('col-lg-3', 'col-md-3', 'hidden-sm', 'hidden-xs'),
+  classNames('col-lg-3', 'col-md-3', 'col-sm-4', 'col-xs-6'),
+];
+
+const UsersTableHeader = () => {
+  return [
+    {
+      title: 'User Name',
+      props: { className: tableColumnClasses[0] },
+    },
+    {
+      title: 'Domain',
+      props: { className: tableColumnClasses[1] },
+    },
+    {
+      title: 'Log in time',
```

Comment:
Use "Login time" instead of "Log in time". There's a slight difference between the two. "Log in" is an instruction to someone to log in  to the system, with "log" being used as a verb. A "login" is a noun and refers to either the credentials used to log in to a system or a specific instance where a user has logged in.

Reply:
hmm.. thanks !
did look a little off to me didn't know why :-)